feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -254,6 +257,33 @@ func (tc *testCase) transferOS(ctx context.Context, t *testing.T, standby bool)
 	}
 }
 
+func (tc *testCase) isSupervisorReady(ctx context.Context, t *testing.T) error {
+	var counter int8
+	const maxRetries = 15
+	for i := int8(1); i < maxRetries; i++ {
+		if counter == maxRetries {
+			return fmt.Errorf("OS.Verify RPC did not report the standby supervisor state as available")
+		}
+		counter++
+		r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+		if err != nil {
+			t.Fatal(err)
+		}
+		switch v := r.GetVerifyStandby().GetState().(type) {
+		case *ospb.VerifyStandby_StandbyState:
+			switch v.StandbyState.GetState() {
+			case ospb.StandbyState_UNSPECIFIED, ospb.StandbyState_UNSUPORTED:
+				t.Fatalf("OS.Verify RPC reported that the standby supervisor is %v", v.StandbyState.GetState())
```

Comment:
I think you want to time.Sleep 1min here too, right?

Reply:
No, if its unspecified or unsupported, I don't see a reason to sleep?